app/views/items/edit.html.erb

```diff
@@ -0,0 +1,15 @@
+<p><%= link_to 'Merchants Index', '/merchants' %></p>
```

Comment:
You have this link for Merchants Index on the top of all of your views. Since this is something you want on all of your pages, like a navbar, this would be something that would be good to put in the body of your application.html.erb before the yield.